test.sh

```diff
@@ -16,6 +16,35 @@ function join_by {
     fi
 }
 
+function uninstall_helm_repo() {
+    repo_name=$1
+    # clusterrole being pesky here so just uninstall in all namespaces
```

Comment:
I made a few changes that should simplify `test.sh` a lot.

While developing, I kept thinking that I had to do everything in the same namespace, but in reality, people would be deploying their monitoring stack on a different namespace.

I switched it to install on default, which now lets us do `helm upgrade --install` without any issues with the Cluster level resources when trying to reinstall again if cleanup was skipped.

That said, there are hanging CRDs that don't get removed by helm uninstall and must be manually deleted by `kubectl delete crd`

Reply:
This is supposed to only work for testing, not on production clusters. But cleanup is important in case someone accidentally runs this on a cluster that's not ephemeral, like `kind`.